iree/compiler/Translation/SPIRV/XLAToLinalg/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+iree_cc_library(
+  NAME
+    LinalgToSPIRV
+  HDRS
+    "MapHloToScalarOp.h"
+  SRCS
+    "XLAToLinalg.cpp"
+  DEPS
+    LLVMSupport
+    MLIRIR
+    MLIRLinalg
+    MLIRPass
+    MLIRStandarOps
+    MLIRSupport
+    MLIRTransforms
+    tensorflow::mlir_xla
+  ALWAYSLINK
```

Comment:
@hanhanW Seems that the `PUBLIC` flag is missing

Reply:
Thanks for catching this. :)